src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -270,6 +270,12 @@ class DockerBuildImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
         buildFile << """
             task verifyImageId {
                 dependsOn buildImage
+
+                doLast {
+                    // Expose issues with getImageId returning null
+                    // https://github.com/bmuschko/gradle-docker-plugin/issues/818
```

Comment:
I propose you annotate the test method with `@Issue` and point to the relevant Github issue as value.

Reply:
Ah neat, I didn't know about this annotation. Will do.